static/app/views/onboarding/targetedOnboarding/components/sidebar.tsx

```diff
@@ -2,17 +2,18 @@ import styled from '@emotion/styled';
 import {motion, Variants} from 'framer-motion';
 import {PlatformIcon} from 'platformicons';
 
+import {PlatformKey} from 'sentry/data/platformCategories';
 import platforms from 'sentry/data/platforms';
 import {IconCheckmark} from 'sentry/icons';
 import {t} from 'sentry/locale';
 import pulsingIndicatorStyles from 'sentry/styles/pulsingIndicator';
 import space from 'sentry/styles/space';
 import {Project} from 'sentry/types';
 import testableTransition from 'sentry/utils/testableTransition';
-import withProjects from 'sentry/utils/withProjects';
 
 type Props = {
   checkProjectHasFirstEvent: (project: Project) => boolean;
+  onboardingPlatforms: {[key in PlatformKey]?: string};
```

Comment:
@Neo-Zhixing I'd add a comment explaining what this prop means. And perhaps update the name to include `project` so it's more clear this maps to a project